tests/grpc/test_servicers.py

```diff
@@ -74,13 +75,48 @@ async def test_model_infer(
         model_infer_request.ClearField("model_version")
 
     prediction = await inference_service_stub.ModelInfer(model_infer_request)
-
     expected = pb.InferTensorContents(int64_contents=[6])
 
     assert len(prediction.outputs) == 1
     assert prediction.outputs[0].contents == expected
 
 
+@pytest.mark.parametrize("settings", [lazy_fixture("settings_stream")])
+@pytest.mark.parametrize(
```

Comment:
what is `sum_model_settings`?

Reply:
It is a fixture (see [here](https://github.com/SeldonIO/MLServer/blob/c57a6ce5352625d4ff67569053b43c1a9c360100/tests/conftest.py#L147-L150))